requirements.txt

```diff
@@ -1,5 +1,6 @@
 ansible-core==2.13.5; python_version >= "3.8"
 ansible==6.5.0; python_version >= "3.8"
+ansible_runner
```

Comment:
Isn't this managed by `poetry` now?  As such, we want to remove the `requirements.txt` and just update the `pyproject.toml`.  Let me know if you want some help with that.

Reply:
This module has not been updated to poetry. I would rather make that change in a separate PR.

Reply:
I also added the dependency to the pyproject.toml. Another PR can switch the Dockerfile to poetry.

Reply:
Why are we not regenerating the requirements.txt file using `poetry`?

Reply:
Basically, you need to do the following:

```
poetry remove ansible
poetry add ansible_runner
poetry update
poetry export > requirements.txt
git add pyproject.toml requirements.txt
git commit -m 'Replace `ansible` with `ansible_runner`'
```

Reply:
FWIW, once `poetry` is properly updated, and the `requirements.txt` file generated, the python module that shows up in the updated `requirements.txt` file is `ansible-runner`, not `ansible_runner`.

Reply:
> Why are we not regenerating the requirements.txt file using `poetry`?

In a following PR we'll remove requirements.txt in favor of exclusively poetry, so this is just temporary. That PR will be the one that switches it to the new format for the Dockerfile, which will build the requirements.txt in a build step.

Reply:
The requirements.txt file is wrong for `ansible_runner`.  We can't separate these changes.  Why can't we just do the above steps and commit this PR?

Reply:
They are functionally distinct changes that I would rather not munge into a single PR. The `requrements.txt` file as provided passes both manual and automated tests. I don't see what stops these from being distinct changes.

Reply:
> ```
> poetry remove ansible
> ```

The ansible plugin is still used for one function.

Reply:
Why does the fact that it passes manual and automated tests say it is working?  If the file contents are incorrect, isn't it possible that the incorrect contents are making it pass?

I'll post a PR with the rebuilt changes so we can evaluate.

Reply:
> FWIW, once `poetry` is properly updated, and the `requirements.txt` file generated, the python module that shows up in the updated `requirements.txt` file is `ansible-runner`, not `ansible_runner`.

I tested this, and both `ansible_runner` and `ansible-runner` in the `requirements.txt` file work with the same result.

Reply:
> Why does the fact that it passes manual and automated tests say it is working? If the file contents are incorrect, isn't it possible that the incorrect contents are making it pass?

I don't understand your question. That is exactly what tests are for.

Reply:
See PR https://github.com/arcalot/arcaflow-plugin-metadata/pull/11.

Reply:
And PR #12.